src/web-rtc-client.js

```diff
@@ -491,6 +491,30 @@ export default class WebRTCClient extends Emitter {
       this.sessionHasRemoteVideo(sessionId);
   }
 
+  sessionHasAudio(session: Session) {
+    const pc = session.sessionDescriptionHandler.peerConnection;
+    let hasAudio = false;
+    if (pc.getSenders) {
+      pc.getSenders().forEach(sender => {
+        if (sender.track && sender.track.kind === 'audio') {
+          if (sender.track.enabled) {
+            hasAudio = true;
```

Comment:
You can `return true` and `return false` at the end of the method. It will save you a variable

Reply:
A `return` inside a `forEach` only ends the current iteration, the function won't receive the returned value. The forEach creates a new scope which captures the return.

https://www.competa.com/blog/the-javascript-array-foreach-method-doesnt-return-anything/

Reply:
Oh sorry, I didn't see the `forEach`, can it be a `for( ... of ... ) ` ?

Reply:
A return inside a `for ... of ...` interrupts the iterator.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Closing_iterators